examples/operator-postgres-tls-demo/01-Install-postgres-tls-using-helm.ipynb

```diff
@@ -0,0 +1,557 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f16967ef",
+   "metadata": {},
+   "source": [
+    "# Deploy PostgreSQL with Helm in TLS Mode"
```

Comment:
any reason we are using PostgreSQL with helm ? I mean we can use similar like -> https://github.com/feast-dev/feast/blob/master/examples/operator-quickstart/postgres.yaml

Reply:
Two main reasons for this approach:

1. Kubernetes Security Constraints – Accessing cert files inside the Postgres container requires bypassing strict Kubernetes security measures, which involves significant logic. I have [exported](https://gist.github.com/lokeshrangineni/4f60aba489969d827206a1f00d59e338) current helm's logic for the comparison with the postgres deployment without TLS.

2. Focus on Feast Go Operator Volumes –  The goal of this example is to demonstrate Feast Go Operator volumes and volumeMounts, not Postgres TLS setup. Offloading TLS management to Helm ensures that it works in future and offloads the managing documentation/any issues by delegating to helm or helm community.